examples/air-quality-sensor-app/linux/AirQualitySensorAppAttrUpdateDelegate.cpp

```diff
@@ -0,0 +1,314 @@
+        &FormaldehydeConcentrationMeasurement::Attributes::MinMeasuredValue::Set,
+        &FormaldehydeConcentrationMeasurement::Attributes::MaxMeasuredValue::Get,
+        &FormaldehydeConcentrationMeasurement::Attributes::MaxMeasuredValue::Set } },
+    { "Pm1ConcentrationMeasurement",
+      { &Pm1ConcentrationMeasurement::Attributes::MeasuredValue::Set,
+        &Pm1ConcentrationMeasurement::Attributes::MinMeasuredValue::Get,
+        &Pm1ConcentrationMeasurement::Attributes::MinMeasuredValue::Set,
+        &Pm1ConcentrationMeasurement::Attributes::MaxMeasuredValue::Get,
+        &Pm1ConcentrationMeasurement::Attributes::MaxMeasuredValue::Set } },
+    { "Pm10ConcentrationMeasurement",
+      { &Pm10ConcentrationMeasurement::Attributes::MeasuredValue::Set,
+        &Pm10ConcentrationMeasurement::Attributes::MinMeasuredValue::Get,
+        &Pm10ConcentrationMeasurement::Attributes::MinMeasuredValue::Set,
+        &Pm10ConcentrationMeasurement::Attributes::MaxMeasuredValue::Get,
+        &Pm10ConcentrationMeasurement::Attributes::MaxMeasuredValue::Set } },
+    { "TotalVolatileOrganicCompoundsConcentrationMeasurement",
+      { &TotalVolatileOrganicCompoundsConcentrationMeasurement::Attributes::MeasuredValue::Set,
+        &TotalVolatileOrganicCompoundsConcentrationMeasurement::Attributes::MinMeasuredValue::Get,
+        &TotalVolatileOrganicCompoundsConcentrationMeasurement::Attributes::MinMeasuredValue::Set,
+        &TotalVolatileOrganicCompoundsConcentrationMeasurement::Attributes::MaxMeasuredValue::Get,
+        &TotalVolatileOrganicCompoundsConcentrationMeasurement::Attributes::MaxMeasuredValue::Set } },
+    { "RadonConcentrationMeasurement",
+      { &RadonConcentrationMeasurement::Attributes::MeasuredValue::Set,
+        &RadonConcentrationMeasurement::Attributes::MinMeasuredValue::Get,
+        &RadonConcentrationMeasurement::Attributes::MinMeasuredValue::Set,
+        &RadonConcentrationMeasurement::Attributes::MaxMeasuredValue::Get,
+        &RadonConcentrationMeasurement::Attributes::MaxMeasuredValue::Set } }
+};
+
+AirQualitySensorAttrUpdateHandler * AirQualitySensorAttrUpdateHandler::FromJSON(const char * json)
+{
+    Json::Reader reader;
+    Json::Value value;
+
+    if (!reader.parse(json, value))
+    {
+        ChipLogError(NotSpecified,
+                     "AllClusters App: Error parsing JSON with error %s:", reader.getFormattedErrorMessages().c_str());
+        return nullptr;
+    }
+
+    if (value.empty() || !value.isObject())
+    {
+        ChipLogError(NotSpecified, "AllClusters App: Invalid JSON command received");
+        return nullptr;
+    }
+
+    if (!value.isMember("Name") || !value["Name"].isString())
+    {
+        ChipLogError(NotSpecified, "AllClusters App: Invalid JSON command received: command name is missing");
+        return nullptr;
+    }
+
+    return Platform::New<AirQualitySensorAttrUpdateHandler>(std::move(value));
```

Comment:
Where is the corresponding Platform::Delete?  This seems to leak the handler....